packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -358,13 +368,19 @@ const renderGraphControls = (props: {
     state: InteractiveGraphState;
     dispatch: (action: InteractiveGraphAction) => unknown;
     width: number;
+    strings: PerseusStrings;
```

Comment:
Instead of adding a prop for the strings, could we call `usePerseusI18n` here too? Or is that not possible / advisable? It seems like using the context might be a bit more convenient.

Reply:
usePerseusI18n throws a lint error if it's not used in a component directly. :(